api/registry/api/v1.py

```diff
@@ -751,3 +753,42 @@ def get_gtc_stake(request, address: str, round_id: int) -> GtcEventsResponse:
         return {"results": [obj for obj in filtered_queryset.values()]}
     except Exception as e:
         raise StakingRequestError()
+
+
+@router.get(
+    "/gtc-stake/{str:address}",
```

Comment:
The return of this API `/gtc-stake/{str:address}` is confusing because it is different from the one in: `/gtc-stake/{str:address}/{int:round_id}` 

I would suggest naming the path like: `/gtc-stake/v2/{str:address}` just to make it clear to the user: we are using a new staking SC and expect other data format here.